### DISCUSSION TITLE
[request] Duplicati

### DISCUSSION THREAD

**Discussion by Supermagnum** [swizzin/swizzin#1096] [Feature Requests]
### What happened?

Duplicati is a backup solution that works with standard protocols like FTP, SSH, WebDAV as well as popular services like Backblaze B2, Tardigrade, Microsoft OneDrive, Amazon S3, Google Drive, box.com, Mega, hubiC and many others. It can be set up to do scheduled backups.
It's found here:
https://www.duplicati.com/

### Swizzin commit

abc

### What OS are you using?

Ubuntu 22.04 (Jammy)

### What architecture is your OS?

amd64

**Comment by raspberryhead** [comment]
+1

**Comment by wii747** [comment]
+1

**Comment by Clumpton** [comment]
+1
Could anyone at least confirm that adding duplicati to a Swizzin-equipped Linux doesn't break anything?
